docs/source/tutorial/create_pipelines.md

```diff
@@ -150,37 +150,6 @@ from kedro.pipeline import Pipeline, node, pipeline
 from .nodes import preprocess_companies, preprocess_shuttles
```

Comment:
The spaceflights tutorial does `kedro run --pipeline=ds` somewhere; this will need to change to `data_science`.

Reply:
@AntonyMilneQB https://github.com/kedro-org/kedro-starters/pull/101, but do we need to worry about people using the starter with older version of Kedro?

Reply:
https://github.com/kedro-org/kedro/commit/36e0708ad2427ef19783c37acb62b8bbd0745122 changes the references in this file.

Reply:
No need to worry because `kedro new -s spaceflights` with kedro version 0.18.2 will pull the starter tagged with 0.18.2 (which has `ds` and `dp` pipelines), and people can look at the 0.18.2 version of the documentation to match. With kedro version 0.18.1, `kedro new -s spaceflights` will pull the starter tagged with 0.18.3 and the new pipeline names that match the updated docs.